lntest/itest/lnd_taproot_test.go

```diff
@@ -272,3 +273,150 @@ func testTaprootScriptSpend(ctxt context.Context, t *harnessTest,
 	// tx is actually included in a block.
 	mineBlocks(t, net, 1, 1)
 }
+
+// testTaprootKeySpendRPC tests that a tapscript address can also be spent using
+// the key spend path through the RPC.
+func testTaprootKeySpendRPC(ctxt context.Context, t *harnessTest,
+	net *lntest.NetworkHarness) {
+
+	// For the next step, we need a public key. Let's use a special family
+	// for this.
+	const taprootKeyFamily = 77
+	keyDesc, err := net.Alice.WalletKitClient.DeriveNextKey(
+		ctxt, &walletrpc.KeyReq{
+			KeyFamily: taprootKeyFamily,
+		},
+	)
+	require.NoError(t.t, err)
+
+	internalKey, err := btcec.ParsePubKey(keyDesc.RawKeyBytes)
+	require.NoError(t.t, err)
+
+	// Let's create a taproot script output now. This is a hash lock with a
```

Comment:
Not a huge blocker, but this fragment shows up a few times in the integration tests and could be moved to a simple routine.